src/server.js

```diff
@@ -1,13 +1,49 @@
-const express = require('express');
-const dotenv = require('dotenv');
+import '@babel/polyfill';
+import morgan from 'morgan';
+import express from'express';
+import { json } from 'express';
+import dotenv from 'dotenv';
+import colors from 'colors';
+import logger from './App/middlewares/logger';
+import routes from './App/routes/index';
 
+
+//Load env vars
+dotenv.config();
+
+//Initialising the app with express
 const app = express();
 
-//Body parser
-app.use(express.json());
+//Initialise logger middleware
+app.use(logger);
+
+//
+app.use(json());
+
+// Mounting Morgan
+app.use(morgan('dev'));
 
+//Mounting routes
+app.use('/', routes);
 
+//DB Initialisation
+const db = require('./Database/models/server');
+
+//Test DB Connection
+db.sequelize.authenticate()
+    .then( () => console.log('Connection has been established successfully...'.bgGreen.bold))
+    .catch( err => console.error('Unable to connect to the database:'.bgRed.bold, err.message));
+
+db.sequelize.sync({ alert: true});
```

Comment:
I really wonder why are you adding this while you know it clears the database!???